st2common/st2common/logging/formatters.py

```diff
@@ -73,7 +74,7 @@ def serialize_object(obj):
     if getattr(obj, 'to_dict', None):
         value = obj.to_dict()
     elif getattr(obj, 'to_serializable_dict', None):
-        value = obj.to_serializable_dict()
+        value = obj.to_serializable_dict(mask_secrets=True)
```

Comment:
Note: I tried really hard and wanted to avoid this and keep all the masking code and related functionality inside the log formatter since this should ideally only be responsibility of the log formatter, but sadly that's not possible so some of the masking code has leaked / spilled into other places as well :/

This makes me quite unhappy, but there is really no way around that and I still did my best to keep leakage to the minium.